lib/deploy.js

```diff
@@ -49,8 +49,11 @@ const deployCLI = async function deployCLI(funcName, funcPath, funcConf) {
     },
   });
   const apiKey = await getAPIKey();
-  await deploy(funcName, apiKey, funcConf, funcTarPath);
-  return urljoin(`https://${getInvokeEndpoint()}`, 'v1', 'run', apiKey, funcName);
+  const { advice } = await deploy(funcName, apiKey, funcConf, funcTarPath);
```

Comment:
why is this only part of deploy? what about remove, log, list?
If this is deploy only feature, no need to print from common `exceptionWrapper`.
If it isn't then why aren't other commands do so too?

Reply:
The other commands have been updated since I submitted this patch. I'm trying to keep this granular and agile. Once I make the same updates to `picard` etc as I did to `spice` I'll make the same changes to other SDK/CLI commands.